index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Project 1: Set up and Mobile Version Skeleton</title>
+
+  <link rel="stylesheet" type="text/css" href="styles.css">
+
+</head>
+<body>
+  <header>
+    <nav>
+      <ul class="nav-list">
+        <li><a id="nav-bar-a" href="">Welcome</a></li>
```

Comment:
- [ ] Kindly get rid of the underline that is being printed into this link. You can use `text-decoration: none` in CSS for this

Reply:
I see that the previous reviewer didn't mention this, and I'm assuming it is because you accidentally introduced the underline after the previous review was made.

Reply:
Thanks, @JuliCarracedo! Just to confirm, is that the only change I'd need to address? I noticed you mentioned "changes", but you only said to remove the underline. 

Thanks in advance!